wallet/src/types.rs

```diff
@@ -19,7 +19,7 @@ pub use witnet_data_structures::{
         Block as ChainBlock, CheckpointBeacon, DataRequestInfo, DataRequestOutput, Epoch, Hash,
         HashParseError, Hashable, Input as TransactionInput, KeyedSignature, OutputPointer,
         PublicKey, PublicKeyHash, PublicKeyHashParseError, RADAggregate, RADRequest, RADRetrieve,
-        RADTally, SuperBlock, ValueTransferOutput as VttOutput,
```

Comment:
I approve this change

Reply:
me too

Reply:
idk

Reply:
I am against renaming types, completely misleading in our case!